src/lib.rs

```diff
@@ -367,11 +364,11 @@ where
     /// the scraper again
     pub fn crawl<TCrawlFunction, TCrawlFuture>(&mut self, fun: TCrawlFunction)
     where
-        TCrawlFunction: FnOnce(Arc<reqwest::Client>) -> TCrawlFuture,
```

Comment:
this (and also `complete`) looks like internal Crawler function, but it is `pub`, so formally it is breaking change.

What's better – make this change (and up crate minor version, I guess?) or leave it as it is (almost, it'll use Arc::new instead of Arc::clone)?